app/pages/learn/01_tutorial/01_your-first-java-app/02_building-with-intellij-idea.md

```diff
@@ -0,0 +1,262 @@
+---
+id: first_app.intellij-idea
+title: Building a Java application in IntelliJ IDEA
+slug: learn/intellij-idea
+type: tutorial
+category: start
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+more_learning:
+  youtube:
+    - H_XxH66lm3U
+    - 70_B2DyM8mU
+toc:
+- Overview
+- Installing IntelliJ IDEA
+- Creating a new project
+- Writing and editing code
+- Running your application
+- Testing
+- Debugging
+- Refactoring code
+- Documenting code
+- Searching and navigating
+- Summary
+description: "Learn how to code, run, test, debug and document a Java application faster in IntelliJ IDEA."
+last_update: 2024-04-04
+author: ["MaritvanDijk"]
+---
+
+## Overview
+An IDE (Integrated Development Environment) allows you to quickly build applications, by integrating a source-code editor with the ability to compile and run your code, as well as integration with  tools you’ll need for software development, including build tools, testing and debugging tools, version control, and so on. And finally, an IDE will let you search and navigate your codebase in ways your file system won’t.
```

Comment:
Here's an idea for @ammbra: If the IDE articles are grouped together in a "series" (or whatever that's called), it would be possible to have an introduction to IDEs in the series page (which lists all the article).